.circleci/config.yml

```
@@ -538,3 +583,19 @@ workflows:
             branches:
               only:
                 - master
+      - pytorch_windows_build_worker_0:
+          name: win_test_worker_0
+      - pytorch_windows_build_worker_1:
+          name: win_test_worker_1
+      - pytorch_windows_build_worker_2:
+          name: win_test_worker_2
+      - pytorch_windows_build_worker_3:
+          name: win_test_worker_3
+      - pytorch_windows_build_worker_4:
+          name: win_test_worker_4
+      - pytorch_windows_build_worker_5:
+          name: win_test_worker_5
+      - pytorch_windows_build_worker_6:
+          name: win_test_worker_6
+      - pytorch_windows_build_worker_7:
+          name: win_test_worker_7
```

Here's what a manual approval would look like:

```suggestion
      - hold:
          type: approval
      - pytorch_windows_build_worker_0:
          name: win_test_worker_0
          requires:
            - hold
      - pytorch_windows_build_worker_1:
          name: win_test_worker_1
          requires:
            - hold
      - pytorch_windows_build_worker_2:
          name: win_test_worker_2
          requires:
            - hold
      - pytorch_windows_build_worker_3:
          name: win_test_worker_3
          requires:
            - hold
      - pytorch_windows_build_worker_4:
          name: win_test_worker_4
          requires:
            - hold
      - pytorch_windows_build_worker_5:
          name: win_test_worker_5
          requires:
            - hold
      - pytorch_windows_build_worker_6:
          name: win_test_worker_6
          requires:
            - hold
      - pytorch_windows_build_worker_7:
          name: win_test_worker_7
          requires:
            - hold
```